### PR TITLE
i4.i4Tools version 8-beta

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106162)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230508051441

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

File: manifests/i/i4/i4Tools/8-beta/i4.i4Tools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: i4.i4Tools
+PackageVersion: 8-beta
+Installers:
+- InstallerUrl: https://url.i4.cn/2MvQruaa
+  Architecture: x86
+  InstallerType: exe
```
Comment (stephengillie) [inline manifests/i/i4/i4Tools/8-beta/i4.i4Tools.installer.yaml:9]:
```suggestion
  Architecture: x64
  InstallerType: portable
```

File: manifests/i/i4/i4Tools/8-beta/i4.i4Tools.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: i4.i4Tools
+PackageVersion: 8-beta
+Installers:
+- InstallerUrl: https://url.i4.cn/2MvQruaa
+  Architecture: x86
```
Comment (stephengillie) [inline manifests/i/i4/i4Tools/8-beta/i4.i4Tools.installer.yaml:8]:
```suggestion
  Architecture: x64
```

**Comment by stephengillie** [comment]
Unfortunately, this package doesn't seem to respond to installer switches.

**Comment by Trenly** [comment]
[Policy] Blocking Issue to exclude this PR from search filters until Interactive-Only is supported

**Pull request closed without merging**